brands/amenity/bicycle_rental.json

```diff
@@ -159,6 +159,21 @@
         "name": "OV-fiets"
       }
     },
+    {
+      "displayName": "Santander Cycles",
+      "id": "santandercycles-fa5175b",
+      "locationSet": {"include": ["gb"]},
+      "matchNames": ["barclays bikes", "barclays cycle hire", "boris bike", "boris bikes"],
```

Comment:
Do these work on brand? Hopefully they can be made to do so if they don't currently?

Reply:
I'm not sure what you mean?

Reply:
Sorry, I mean do matchNames just work on a name field or will they also cover brands, or is that a question for iD and other consumers?

i.e. if I type in Boris Bike will if offer Santander Cycles?

Reply:
> i.e. if I type in Boris Bike will if offer Santander Cycles?

Yes, the brand tag is considered an alternate name also.